### DISCUSSION TITLE
The new Beta feed (For You) is presenting people that I don't want to see.

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#13183] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/1277920/159373185-bac8be5e-d99b-4b2c-975f-7731306a9637.png)

I just follow 4 people. I am receiving recommendations for people that I don't follow. Please review this thing. I don't want to see random people that I don't want to see.

Please, don't turn this wonderful platform into another Facebook. The goal here is to share code. Not to "socialize". 

At least give us something, that is easy to use, to hide/block/disable/remove/purge people from the feed.

Honestly, I already left services for way less...

**Comment by coderhs** [comment]
I would like to second that, a small cancel or cross next to the name would also be enough. Its recommending people i actually would like to block from my life.

**Reply by knownasilya** [reply]
Was just about to post this recommendation. Like an "ignore".

**Reply by leoheck** [reply]
Exactly. A simple, single, unique "Ignore this person" will be more than enough. And maybe a way to revert it somewhere... because, you know, we are stupid sometimes.

**Comment by leoheck** [comment]
Now, I am currently following 0 (zero) people, just because of this feed. I removed all the ones I was following, just because of this, and it did not improve anything. There are people and content I don't want to see. You just have to allow us to stop pushing them to my feed. This is not social media. If Github is turning into another Facebook, then let us know, so we can leave and remove our projects from this website.

**Reply by coderhs** [reply]
I created this userscript to hide the recommended list, until github allows me to not show people to me.

https://github.com/coderhs/hide-github-recommended-for-you

**Reply by leoheck** [reply]
that's awesome, thanks @coderhs

**Comment by twobiers** [comment]
You can disable recommendations in the filter options. 
![image](https://user-images.githubusercontent.com/22715034/177728486-ded7bdc7-8564-47f6-bcbe-6f7591da3153.png)

**Reply by coderhs** [reply]
Thanks for this details, I wans't aware of the filter.

Sadly it removes the repo recommendations as well. Its the who to follow that I would like to eliminate.

**Reply by leoheck** [reply]
> You can disable recommendations in the filter options.

I don't want to disable recommendations. The recommendations are good. This issue here is to ask for an option to HIDE/REMOVE unwanted people/things from this window.

**Comment by maska228** [comment]
chad hikikomori dev doesnt want to "socialize" [sic.]

**Reply by leoheck** [reply]
@maska228 I have no idea who is this chad hikikomori dev, sorry.

**Reply by maska228** [reply]
> @maska228 I have no idea who is this chad hikikomori dev, sorry.

that was a joke, that seemed to fly over your head because my humor is dumb af

**Reply by leoheck** [reply]
that's fine, it looks like you didn't get my humor either.

**Reply by hai284** [reply]
> chad hikikomori dev không muốn "xã hội hóa" [sic.]

theo cách củ 1 ngón tay

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by leoheck** [reply]
Hello jdevstatc. This setggins does not do anything useful. 

I don't want to stop seeing these things. I am just not interesting to have feedback about some people eating, talking, sneezing, pooping, e.v.e.r.y. time. 

I also don't want to stop any chance this person has to see my stuff or interact with me. 

If they interact with me, or me with them, then this is important. Otherwise, it is just useless garbage flowing e.v.e.r.y time. 

So, I am asking for a way to stop seeing this useless feedback from people that I don't want to see. But I don't want to jail the person. Especially co-workes. I don't care about the torrent of useless information that this thing is generating e.v.e.r.y time. 

However, I do like to receive news of new stuff. This is really good. And I also want to be in control of the shit that is coming. Otherwise, Github is just creating another useless platform like the Television advertisement that came in the '80s that e.v.e.r.y one hate and it is dying these days.

**Reply by jfullstackdev** [reply]
is that not part of Recommendations ? only check Announcements if you want 

with the last resort, block those users

**Comment by knownasilya** [comment]
This is part of a solution. In my case I want to see people  recommendations, but don't want to keep seeing the same recommendations for people if I don't want to follow them. So closing individual recommendations would be a much better solution.

**Reply by leoheck** [reply]
Exactly. Who is driving this platform these days? I feel they are just blindfolded and mature and have never used the internet ever. I can only think that.

**Reply by jamieshark** [reply]
I promise we have used the internet before. Some of us, in fact, were raised alongside it. We are trying our best to balance what users are asking for and what we have capacity to allocate our efforts to. Thanks for your patience and feedback!

**Reply by jfullstackdev** [reply]
yup that's right, as I'm always telling them too, we need to consider all users

**Reply by leoheck** [reply]
> I promise we have used the internet before. Some of us, in fact, were raised alongside it. We are trying our best to balance what users are asking for and what we have capacity to allocate our efforts to. Thanks for your patience and feedback!

That was an awesome answer. Than you @jamieshark
